app/pom.xml

```diff
@@ -322,6 +322,7 @@
         <basepom.check.skip-all>true</basepom.check.skip-all>
         <basepom.it.skip>true</basepom.it.skip>
         <skipTests>true</skipTests>
+        <maven.test.skip>true</maven.test.skip>
```

Comment:
Not sure we need this, compiler, resource, surefire and failsafe are removed from running below, so this doesn't make a difference.

Reply:
Thought this might solve the problem but sadly not.